Source/WebSocket/hcwebsocket.cpp

```diff
@@ -373,6 +388,12 @@ HRESULT HC_WEBSOCKET::SetHeader(
     return S_OK;
 }
 
+HRESULT HC_WEBSOCKET::SetMaxReceiveBufferSize(size_t maxReceiveBufferSizeBytes) noexcept
+{
+    m_maxReceiveBufferSize = maxReceiveBufferSizeBytes;
+    return S_OK;
```

Comment:
is this used anywhere?

Reply:
its returned to clients via HCWebSocketSetMaxReceiveBufferSize. Currently this method can't fail but I think its fine to keep this signature in case that changes in the future